src/js/cbLink.jsx

```diff
@@ -0,0 +1,12 @@
+var React = require('react');
```

Comment:
Just curious, do we *need* this component? Couldn't we just render links as normal with JSX from within the other components?

Doing it this way, should we use this component to render all links in the app?